### ISSUE TITLE
Unclear installation instructions for Nvim

### ISSUE THREAD

**Issue by jonatkinson** [zxqfl/tabnine-vim#4]
The installation instructions should display a method of validating whether the plugin has installed successfully; I've installed the plugin using `vim-plug`, as follows:

	" TabNine complation
        Plug 'zxqfl/tabnine-vim'

The plugin successfully installs in my environment (as reported by `vim-plug`, at least), but I can't run anything in the `:TabNine::*` namespace (for example, `TabNine::version`).

Based issue #2, I've also run `install.py` in the plugin folder, but it's unclear what this did (I'm not particularly familiar with YouCompleteMe, either).

**Comment by dteoh** [comment]
I'm using nvim 0.3.1 with vim-plug. Set up python integration (`:help provider-python` for details), install the plugin, open some source code file, press "Tab" in edit mode and see that it works.

**Comment by ivanovsaleksejs** [comment]
It actually shows me an error:
`Can't start TabNine. Tried the following paths: ['/home/aleksejs/.config/nvim/plugged/tabnine-vim/.../.config/nvim/plugged/tabnine-vim/python/ycm/../../binaries/0.6.0/i686-pc-windows-gnu/TabNine.exe']`

**Comment by dteoh** [comment]
Okay... so the TabNine commands are literally to be typed in a buffer, not as a command.

Open an empty file, go into insert mode, and type "TabNine::version", and then you see an autocomplete with the version number in it.

This should probably be a command.

**Comment by jonatkinson** [comment]
Yep, this clarifies things (and it does appear my installation is correct). I suspect a documentation patch would be wise as I won't be the only one who expected a Vim command.

**Comment by halas** [comment]
I was also expecting vim command :) would be great to clarify this in documentation, as this is really unexpected.
